tasks/Functions-client/setAutoRequest.js

```diff
@@ -44,92 +50,74 @@ const setAutoRequest = async (contract, taskArgs) => {
     throw Error("Gas limit must be less than or equal to 300,000")
   }
 
-  console.log(`Setting the Functions request in AutomatedFunctionsConsumer contract ${contract} on ${network.name}`)
+  console.log(`\nSetting the Functions request in AutomatedFunctionsConsumer contract ${contract} on ${network.name}`)
 
   const autoClientContractFactory = await ethers.getContractFactory("AutomatedFunctionsConsumer")
   const autoClientContract = await autoClientContractFactory.attach(contract)
 
   const unvalidatedRequestConfig = require(path.isAbsolute(taskArgs.configpath)
     ? taskArgs.configpath
     : path.join(process.cwd(), taskArgs.configpath))
+
   const requestConfig = getRequestConfig(unvalidatedRequestConfig)
 
-  // doGistCleanup indicates if an encrypted secrets Gist was created automatically and should be cleaned up by the user after use
-  let doGistCleanup = !(requestConfig.secretsURLs && requestConfig.secretsURLs.length > 0)
-  const request = await generateRequest(requestConfig, taskArgs)
+  console.log("\nEncrypting secrets and uploading to DON...")
```

Comment:
Do you also want to support remote secrets or do you prefer to exclusively use DON hosted?
While it might be nice to give users both options, I do see risk that users won't remember to delete their Gists...

If we choose not to support `Location.Remote` in this script, then we should give some sort of error if the user specifies `Location.Remote` in Functions-request-config.js

Reply:
Great catch!